upgrading/upgrade-helm.adoc

```diff
@@ -7,15 +7,23 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-You can upgrade to the latest version of {product-title} from a supported older version. For upgrading to {product-title-short} 4.0, you must be using the latest patch release of {product-title-short} 3.74. If you are using an older version, you must first upgrade to {product-title-short} 3.74.
+Upgrading {product-title-short} requires a specific upgrade path depending on the release of {product-title-short} that you are running. You must also back up your Central database before updating the Helm chart and peforming the upgrade.
 
-If you have installed {product-title} by using Helm charts, to upgrade to the latest version of {product-title} you must perform the following:
+[id="upgrading-from-374_{context}"]
+== Upgrade sequence from {product-title-short} release 3.74 and earlier
 
-* Backup the Central database.
-* (Optional) Optimize Central database and Persistent Volume Claims (PVC).
-* (Optional) Generate `values-private.yaml` configuration file containing root certificates for the central-services Helm chart.
-* Update the Helm chart.
-* Run the `helm upgrade` command.
+When upgrading from earlier releases, follow this guidance:
+
+* If the release for Central is earlier than 3.74, you must upgrade to the latest 3.74 patch before upgrading to a 4.x release. See the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74] for information about upgrades from earlier versions to 3.74.
+* When upgrading Helm-based installations from release 3.74, you can upgrade to any latest patch of {product-title-short} version 4.0 through 4.4. However, for full functionality, upgrade to release 4.4.
+
+If you have installed {product-title-short} by using Helm charts, to upgrade to the latest version of {product-title-short} perform the following steps:
+
+. Back up the Central database.
+. (Optional) Optimize Central database and Persistent Volume Claims (PVC).
+. (Optional) Generate `values-private.yaml` configuration file containing root certificates for the central-services Helm chart.
```

Comment:
```suggestion
. Optional: Optimize central database and Persistent Volume Claims (PVC).
. Optional: Generate `values-private.yaml` configuration file containing root certificates for the central-services Helm chart.
```
or, because it's not a procedure, maybe: 
```suggestion
. Optionally, optimize central database and Persistent Volume Claims (PVC).
. Optionally, generate `values-private.yaml` configuration file containing root certificates for the central-services Helm chart.
```

Reply:
Central is a component and is always capitalized in the RHACS docs.